src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,7 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
```

Comment:
this method should call `findByLogin()` and if user exists check if passwords match